components/ActionButtons/ActionButtons.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Link, Stack } from "@chakra-ui/react";
+import { SignupButton } from "components/Homepage/SignupButton";
+import { useNavOptions } from "components/hooks/useNavOptions";
+
+export const ActionButtons = () => {
+  const { howItWorks } = useNavOptions();
```

Comment:
nit: I noticed that `useNavOptions` returns a static object - which doesn't actually make it a hook - and it could arguably just be an exported const instead. Just an FYI!

Reply:
Good catch. Moved to an enum